sdk/eventhubs/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/ReactorSender.java

```diff
@@ -353,7 +353,14 @@ private void processDeliveredMessage(Delivery delivery) {
                 }
             }
 
-            final Duration retryInterval = retry.getNextRetryInterval(exception, workItem.timeoutTracker().remaining());
+            Duration retryInterval = null;
+
+            if (exception instanceof AmqpException && AmqpExceptionUtil.isRetriableException(exception)) {
```

Comment:
What if the criteria for a retriable exception is not standard for all developers, and there's a desire to use different logic in a custom retry policy?   

An example from the top of my head:  I have my throughput units low and have a bunch of concurrent activity in my application.  I know that I'm going to see a bunch of timeout exceptions until the burst of activity stops.   If I see one, I want to use a custom retry policy to wait a longer-than-normal amount of time in order to break the cycle.